Course.php

```diff
@@ -353,18 +355,18 @@
           
           <div id="menu4" class="container tab-pane"><br>
          <?php
-      
+      $group_id=$_SESSION['group_id'];
 $resultx  = mysqli_query($con,"SELECT `Submission_ID`, `Submission_Date`, lab_reports_table.`Lab_Report_ID`, `Student_id`, "
         . "`Course_Group_id`, `Notes`, lab_report_submissions.`Marks`,
         lab_report_submissions.Remarking_Reason,
         `Status`, lab_reports_table.Title Lab_Title,lab_reports_table.Marks Original_marks FROM `lab_report_submissions` "
         . "INNER JOIN lab_reports_table on lab_reports_table.Lab_Report_ID=lab_report_submissions.Lab_Report_ID "
-        . "WHERE lab_report_submissions.Student_id='$student_id' and" 
+        . "WHERE lab_report_submissions.Student_id='$student_id' or lab_report_submissions.Course_Group_id='$group_id'and" 
         . ""
         . ""
         . ""
         . " lab_reports_table.Lab_Report_ID  in (select Lab_Report_ID from lab_report_submissions"
-         . " where  (Status='Marked' or Status='Remarking') and (Student_id=$student_id)  and Course_ID=$course_id) ORDER by Submission_ID DESC");
+         . " where  (Status='Marked' or Status='Remarking') and (Student_id=$student_id or Course_Group_id=$group_id)  and Course_ID=$course_id) ORDER by Submission_ID DESC");
```

Comment:
@Teecloudy 

Cannot under this nested SQL statement in 10 seconds.  

That means it needs a higher-level comment to describe the purpose of this statement.

**Hui**

Reply:
> @Teecloudy
> 
> Cannot under this nested SQL statement in 10 seconds.
> 
> That means it needs a higher-level comment to describe the purpose of this statement.
> 
> **Hui**

About the 10 seconds i didnt fully understand but i added "or Course_Group_id=$group_id" to make sure all other group members can be informed once the instructor marks their assignment.

Reply:
@Teecloudy 

Sorry.  It should be "understand" not "under".   When I said "in 10 seconds", I meant in short time.  I could not understand the SQL statement in short time without the help from comments.

**Hui**